### PR TITLE
Adding stactools recipe

### PR DESCRIPTION
**Pull request by geomatician**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by geomatician** [comment]
@conda-forge/help-python The recipe has passed all checks and is ready for your review. Thanks.

**Comment by gadomski** [comment]
Dropping by to say that I am 🆗 with being the maintainer for this recipe.

File: recipes/stactools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    - requests >=2.20
+    - shapely >=1.6
+    - stac-check >=1.2.0
+    - stac-validator >=3.1.0
+
+test:
+  imports:
+    - stactools
+  commands:
+    - pip check
```
Comment (beenje) [inline recipes/stactools/meta.yaml:42]: Could you please add `stac --help` to the tests?

File: recipes/stactools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/stactools-{{ version }}.tar.gz
+  sha256: 691dc0038d3e709462748747d2e39c2f369cf16d5cd890faba0025e4424b4991
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+  - stac = stactools.cli:cmd
```
Comment (beenje) [inline recipes/stactools/meta.yaml:17]: Is this correct? It is `stactools.cli.cli:run_cli` in [setup.cfg](https://github.com/stac-utils/stactools/blob/1f565873604a337d376314b1b8e38241802a94e8/setup.cfg#L62). The best is to add a test to ensure the entry point works as expected.

File: recipes/stactools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  home: https://github.com/stac-utils/stactools
+  summary: Command line tool and Python library for working with STAC
+  license: Apache-2.0
+  license_file: LICENSE
+  doc_url: https://stactools.readthedocs.io/en/stable/
+  dev_url: https://github.com/stac-utils/stactools
+
+extra:
+  recipe-maintainers:
+    - gadomski
```
Comment (beenje) [inline recipes/stactools/meta.yaml:56]: Just to check: @geomatician you don't won't to be a maintainer?

**Review by beenje — CHANGES_REQUESTED** [review]

File: recipes/stactools/meta.yaml (hunk 3 above)
Comment (geomatician) [reply recipes/stactools/meta.yaml:56]: Yes, that is correct.

File: recipes/stactools/meta.yaml (hunk 2 above)
Comment (geomatician) [reply recipes/stactools/meta.yaml]: Corrected

File: recipes/stactools/meta.yaml (hunk 1 above)
Comment (geomatician) [reply recipes/stactools/meta.yaml:42]: Added

**Comment by geomatician** [comment]
@beenje can you please re-review?

**Pull request merged**